### PR TITLE
CC-26541 Introduced automated tests for shipment types SAPI.

### PR DESCRIPTION
**Pull request by ievgen-malykhin**
Ticket: https://spryker.atlassian.net/browse/CC-30625

### CODE REVIEW

**Review by Nataliia-Popkova — APPROVED** [review]

**Comment by Nataliia-Popkova** [comment]
As we discussed with the developer. First needs to be merged PR https://github.com/spryker/robotframework-suite-tests/pull/549 than fresh master should be merged to current and documentation API KEYWORD need to be gnerated.

File: resources/steps/shipment_type_steps.robot [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+    IF    '${uuid}' != '${None}'
+         ${query}    Set Variable    ${query} WHERE uuid = '${uuid}'
+    END
+    Log   ${query}
+    Connect to Spryker DB
+    Execute Sql String    ${query};
+    Disconnect From Database
+    Trigger publish trigger-events    shipment_type    ${console_path}    timeout=1s
+
+Active shipment types
```
Comment (dmiseev) [inline resources/steps/shipment_type_steps.robot:25]:
```suggestion
Activate shipment types
```

File: tests/api/suite/glue/shipment_type_endpoints/shipment_types/positive.robot [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+    Then Response status code should be:    200
+    And Response reason should be:  OK
+    And Each array element of array in response should contain property:    [data]    id
+    And Each array element of array in response should contain property:    [data]    type
+    And Each array element of array in response should contain property:    [data]    attributes
+    And Each array element of array in response should contain property:    [data]    links
+    And Each array element of array in response should contain nested property:    [data]    [attributes]    name
+    And Each array element of array in response should contain nested property:    [data]    [attributes]    key
+    And Each array element of array in response should contain nested property:    [data]    [links]    self
+    Each array element of array in response should contain nested property with value:    [data]    [type]    shipment-types
```
Comment (dmiseev) [inline tests/api/suite/glue/shipment_type_endpoints/shipment_types/positive.robot:40]: duplicated checks, you already check the structure

**Review by dmiseev — APPROVED** [review]

**Comment by dmiseev** [comment]
Closed, provided these tests in codeception.

**Pull request closed without merging**